wordcloud/wordcloud.py

```diff
@@ -538,7 +540,13 @@ def generate_from_text(self, text):
         -------
         self
         """
-        words = self.process_text(text)
```

Comment:
you don't need to change the wordcloud class. You can just use ``generate_from_frequencies``.